Emby.Server.Implementations/Networking/NetworkManager.cs

```diff
@@ -380,15 +509,7 @@ private IEnumerable<IPAddress> GetIPsDefault(bool ignoreVirtualInterface)
             {
                 var ipProperties = network.GetIPProperties();
 
-                // Try to exclude virtual adapters
-                // http://stackoverflow.com/questions/8089685/c-sharp-finding-my-machines-local-ip-address-and-not-the-vms
-                var addr = ipProperties.GatewayAddresses.FirstOrDefault();
-                if (addr == null
-                    || (ignoreVirtualInterface
-                        && (addr.Address.Equals(IPAddress.Any) || addr.Address.Equals(IPAddress.IPv6Any))))
-                {
-                    return Enumerable.Empty<IPAddress>();
-                }
+                // Exclude any addresses if they appear in the LAN list in [ ]
```

Comment:
where does this happen?

Reply:
Good catch! Errant comment - should be removed. Filtering is done in the calling function.